### PR TITLE
OCP4/PCI-DSS: Update section-8

### PR DESCRIPTION
**Pull request by lmzuccarelli**
#### Description:

- Update PCI-DSS Section 8

#### Rationale:

- Work torwards completing PCI-DSS compliance

### CODE REVIEW

File: controls/pcidss_ocp4.yml [hunk 1]
```diff
@@ -1710,45 +1710,70 @@ controls:
 
 - id: Req-8.1
   title: '8.1 Define and implement policies and procedures to ensure proper user identification
-    management for non- consumer users and administrators on all system components
+    management for non-consumer users and administrators on all system components
     as follows:'
   levels:
   - base
   notes: ''
```
Comment (JAORMX) [inline controls/pcidss_ocp4.yml:1717]: could you add a quick note on why this is not applicable?

File: controls/pcidss_ocp4.yml [hunk 2]
```diff
@@ -1938,7 +2013,8 @@ controls:
...
-  status: pending
+
```
Comment (JAORMX) [inline controls/pcidss_ocp4.yml:2016]: could you add a quick note on why this is not applicable?

File: controls/pcidss_ocp4.yml [hunk 3]
```diff
@@ -1952,7 +2028,7 @@ controls:
   levels:
   - base
   notes: ''
-  status: pending
+  status: not applicable
```
Comment (JAORMX) [inline controls/pcidss_ocp4.yml:2031]: could you add a quick note on why this is not applicable?

File: controls/pcidss_ocp4.yml [hunk 4]
```diff
@@ -1961,7 +2037,7 @@ controls:
...
```
Comment (JAORMX) [inline controls/pcidss_ocp4.yml:2040]: could you add a quick note on why this is not applicable?

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: applications/openshift/authentication/idp_is_configured/rule.yml [hunk 5]
```diff
@@ -59,7 +59,7 @@ references:
   cis@ocp4: 3.1.1
   nerc-cip: CIP-004-6 R2.2.2,CIP-004-6 R2.2.3,CIP-007-3 R.1.3,CIP-007-3 R5,CIP-007-3 R5.1,CIP-007-3 R5.1.1,CIP-007-3 R5.1.2,CIP-007-3 R5.1.3,CIP-007-3 R5.2,CIP-007-3 R5.2.1,CIP-007-3 R5.2.3,CIP-007-3 R5.3.1,CIP-007-3 R5.3.2,CIP-007-3 R5.3.3
   nist: AC-2,AC-2(1),AC-2(2),AC-2(3),AC-2(4),AC-2(5),AC-2(6),AC-2(7),AC-2(8),AC-7
-  pcidss: Req-2.2
+  pcidss: Req-2.2, Req-8.1.1
```
Comment (JAORMX) [inline applications/openshift/authentication/idp_is_configured/rule.yml:62]: This makes the build fail with:
```
 ValueError: Comma-separated 'pcidss' reference in /content/products/ocp4/../../applications/openshift/authentication/idp_is_configured/rule.yml contains whitespace. 
 ```
Let's remove the whitespace.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: applications/openshift/authentication/idp_is_configured/rule.yml (hunk 5 above)
Comment (lmzuccarelli) [reply applications/openshift/authentication/idp_is_configured/rule.yml:62]: Ok will do

**Comment by lmzuccarelli** [comment]
@JAORMX - thanks for the review

**Review by JAORMX — APPROVED** [review]
LGTM I'll merge it once CI passes

File: controls/pcidss_ocp4.yml [hunk 6]
```diff
@@ -1799,17 +1840,24 @@ controls:
     * Something you are, such as a biometric.
   levels:
   - base
-  notes: ''
-  status: pending
+  notes: |-
+    The type of authenticators to be used (for example, password or passphrase,
+    token device or smart card, or biometrics) are also managed externally
+    to OpenShift by the identity provider
+  status: not appplicable
```
Comment (JAORMX) [inline controls/pcidss_ocp4.yml:1847]: There's a typo here, and it makes the build break:
```
 RuntimeError: Unable to parse controls from /content/controls/pcidss_ocp4.yml: The given status 'not appplicable' in the control 'Req-8.2' was invalid. Please use one of the following: ['pending', 'planned', 'not applicable', 'inherently met', 'documentation', 'partial', 'supported', 'automated'] 
 ```

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: controls/pcidss_ocp4.yml (hunk 6 above)
Comment (lmzuccarelli) [reply controls/pcidss_ocp4.yml:1847]: Ah my bad - I'll fix it right away

**Review by JAORMX — APPROVED** [review]

**Pull request merged**